src/nomad_measurements/xrd/__init__.py

```diff
@@ -1 +1,16 @@
-from .schema import *
+
+from nomad.config.models.plugins import SchemaPackageEntryPoint
+
+
+class XRDSchemaPackageEntryPoint(SchemaPackageEntryPoint):
+
+    def load(self):
+        from nomad_measurements.xrd.schema import m_package
+
+        return m_package
+
+
+schema = XRDSchemaPackageEntryPoint(
+    name='XRDSchema',
```

Comment:
In the transmission case we space separated this. Not sure what is best but we should keep it consistent I guess.